### ISSUE TITLE
Simulate the workflow like UE4 Blueprint?

### ISSUE THREAD

**Issue by PabloPicose** [paceholder/nodeeditor#316]
Hi there, first of all I´ve followed this lib in the last days so, congrats, incredible code and readability :)

Is there any form to "launch" the code without launch the Widget system? (I dont know the meaning, not sure if "deferred" is the correct word, im not english so... xD )
I´m not sure if I can launch the "compute" of each custom NodeData connected and not show the Widget NodeData base class via FlowScene, like Unreal Engine 4, where you can create the logic of an entity and launch it in the background.
Thanks!

**Comment by paceholder** [comment]
Hi Pablo,

I understand what you mean, this is a so-called "headless mode". 
I have implemented it in the version 3 which is currently located in the branch `v3alpha`.
There I implemented a general-purpose graph visualizing classes. The whole graph is represented by a single class inherited from AbstractGraph (or called similarly). It's up to you to implement the class and wrap your graph backend str ucture.
Alternatively, I still support "the old way" of using the library, i.e. when you write small sub-models per-node. 

I can go into details or explain some pieces of the code, but first checkout the branch and look at the examples. The headless mode is not shown there, but the idea is simply to not create the scene.

Let me know if you have any further questions.

Best, 

Dimitry
